templates/aws-stack.yml

```diff
@@ -116,6 +112,12 @@ Parameters:
     Type: String
     Default: ""
 
+  BuildkiteAgentToken:
```

Comment:
I just tested switching to an ssm parameter, the old BuildkiteAgentToken does stick around in the CloudFormation template due to NoEcho. To remove it I had to redeploy the existing template on the console and uncheck "use existing value" for this parameter.